examples/README.rst

```diff
@@ -13,3 +13,14 @@ series (magnitude and phase images) and respiratory trace information obtained f
 multi-gradient echo (MGRE) magnitude image is used to generate an ROI and resample the static and dynamic Gx, Gy, Gz
 component maps to match the MGRE image. Lastly the average Gx, Gy, Gz values within the ROI are computed for each
 slice.
+
```

Comment:
Rename this file to getting_started --> do this in another PR, pls open an issue